bucket/imo.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.2.29.0",
+    "description": "Imo is a free, simple, and faster international video call & instant messaging app. (With tons of ads ofcourse)",
+    "homepage": "https://imo.im/",
+    "license": "Proprietary",
```

Comment:
`Proprietary` is for when it costs money, and `Freeware` for when it's free to use. Which is appropriate here?

Reply:
freeware

Reply:
but it contains horrible ads

Reply:
Still Freeware, ads not means it is Proprietary

Reply:
Sorry I thought the maintainers will change license